CHANGELOG.md

```diff
@@ -1,8 +1,17 @@
-# [5.0.0]
-upgraded to babelify@10
+# Changelog
 
-<a name="3.0.0"></a>
-# [3.0.0](https://github.com/kumavis/json-rpc-engine/compare/v2.2.3...v3.0.0) (2017-04-04)
+All notable changes to this project will be documented in this file.
 
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
 
+## [5.2.0] - 2020-07-24
+
+### Changed
+
+- Remove `async` and `promise-to-callback` dependencies
+  - These dependencies were used internally for asynchronous control flow.
+  They have been replaced with Promises and native `async`/`await`.
+  This has introduced subtle timing differences during middleware execution, but none that should affect consumers.
```

Comment:
Is this still true? 🤔

Reply:
The timing differences? I believe so; overall, it appears to be faster in terms of event loop ticks.

Reply:
That's why e.g. https://github.com/MetaMask/metamask-extension/pull/9064 had to happen.

Reply:
Ah I see, OK. I thought that was a result of some previous iteration of the async/await branch.